playbooks/group_vars/prometheus/public.yml

```diff
@@ -161,7 +164,17 @@ prometheus_rules:
       annotations:
         summary: "MongoDB instance [[ $labels.node ]] ([[ $labels.instance ]]) down."
         description: "[[ $labels.node ]] ([[ $labels.instance ]]) of job [[ $labels.job ]] is down."
-
+  - name: "OcimQueueSizeRule"
+    rules:
+    - alert: "ocim_queue_too_big"
+      expr: 'redis_key_size{key="{{ OCIM_QUEUE_REDIS_KEY }}"} > {{ PROMETHEUS_ALERT_OCIM_QUEUE_SIZE_THRESHOLD }}'
+      for: "0s"
+      labels:
+        severity: "critical"
+        environment: "[[ $labels.environment ]]"
+      annotations:
+        summary: "Ocim queue overloaded."
+        description: "Ocim queue overloaded."
```

Comment:
Similar to https://github.com/open-craft/ansible-playbooks/pull/177/files#r487199299

Reply:
@lgp171188 while I got you point with moving this elsewhere, I can't find a good way to move this rule.

Would something like assigning anchors to rules, and then overriding `prometheus_rules` with adding of a new rule in `host_vars` acceptable?

Reply:
@0x29a @lgp171188 I don't think we need to remove the rule from here, this is the main alert configuration place. 
We eventually want to add other monitors here (`ocim-exporter`, `edxapp-exporter`), so it doesn't make much sense to keep this too generic, specially since it's a monitoring system specific to our infrastructure.

Reply:
The alternative option would be to move all the rules to `ansible-secrets` but they are not really secret right?

Reply:
@giovannicimolin, @0x29a, I am okay with leaving this as-is without trying to move it somewhere else to keep this generic because it is not straightforward and doesn't give a lot of benefit. 👍🏼